lib/util/tryPackageJson.js

```diff
@@ -3,9 +3,22 @@
 const path = require('path');
 const loadJsonFile = require('load-json-file');
 
-function tryPackageJson(dir, prop) {
+function tryPackageJson(dir, props) {
     return loadJsonFile(path.join(dir, 'package.json'))
-    .then((json) => !!json[prop], (err) => {
+    .then((json) => {
```

Comment:
Instead of "magically" checking the dependencies, what if we simply do `['dependencies.standard', 'devDependencies.standard']` for standardjs?

This means that this function becomes simpler and there's less magic.

Reply:
I like that approach. I can add these updates when I'm home from work